src/modules/atomic-queue/IAtomicSolver.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.8.0;
+
+import { ERC20 } from "@solmate/tokens/ERC20.sol";
+
+interface IAtomicSolver {
```

Comment:
natspec has been added. `runData` is very open ended, and is fully dependent on how the solver contract is setup, so there is really no example `runData` because it will change for every solver